.travis.yml

```diff
@@ -42,8 +40,6 @@ matrix:
       python: 3.5
     - env: TOX_SUFFIX="flakes"
       python: pypy
-    - env: TOX_SUFFIX="flakes"
-      python: pypy3
```

Comment:
this pypy3 should be pypy3.5-5.9.0

Reply:
Will update these. Should the vanilla `- pypy` be removed?

Reply:
dunno. I'm tempted to have both. @kevin1024 ^

Reply:
And should `python: pypy` be replaced with `python: "pypy2.7-5.8.0"`?

Reply:
oh I'm dumb this is under 'exclude' it shouldn't be here.

Reply:
> And should python: pypy be replaced with python: "pypy2.7-5.8.0"?

I can't decide it if `pypy` should be replaced with `pypy2.7-5.8.0` or `pypy2.7-5.8.0` should be added in addition to the existing plain `pypy` @kevin1024 ^

Reply:
I'd suggest just one, and whichever is the latest. The build matrix is already huge and takes hours to run.

In fact, they're both the same at the moment:

`pypy` => PyPy 5.8.0
`pypy2.7-5.8.0` => PyPy 5.8.0

I would probably go for just `pypy`, as it will be upgraded at some point.

Reply:
oh cool, I thought pypy was still the built in Ubuntu one. OK I'll go with your judgment on this.

Reply:
@graingert I've removed `pypy2.7-5.8.0` and left the more general `pypy`. 

Should `pypy3.5-5.9.0` also be excluded from `TOX_SUFFIX="flakes"`? Everything else but 3.6 is. 

`pypy` is also excluded from `TOX_SUFFIX="aiohttp"`. Should `pypy3.5-5.9.0` be excluded too?

Reply:
> Should pypy3.5-5.9.0 be excluded too?

pypy3.5 should support aiohttp.

Reply:
Yep. And pypy3.5 with flake?